buildlib/azure-pipelines.yml

```diff
@@ -136,6 +136,25 @@ stages:
               ./contrib/buildrpm.sh -s -t -b
             displayName: Build tarball
 
+      # Publish JUCX to maven central
+      - job: publish_jucx
+        displayName: Publish JUCX SNAPSHOT artifact to maven central
+        container: fedora
+        condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+        steps:
+          - bash: ./autogen.sh
+            displayName: Setup autotools
+
+          - bash: |
+              set -eE
+              ./contrib/configure-release --with-java
+              make -s -j`nproc`
+            displayName: Configure
```

Comment:
it also builds (make ...), why displayname is "Configure"?

Reply:
Just copied existing step: https://github.com/openucx/ucx/blob/master/buildlib/azure-pipelines.yml#L115

Reply:
Fixed